web3-dev/decentralized-web.rst

```diff
@@ -6,90 +6,64 @@
 Massa's decentralized web
 #########################
 
-Hosting your website on Massa's decentralized web
-=================================================
 
-Massa's decentralized web allows you to store websites directly on the blockchain.
-This feature enables a lot of applications and provides another layer of security to your dApps.
+Massa’s decentralized web allows you to store websites directly on the blockchain. This feature enables a lot of applications and provides another layer of security to your dApps.
 
-This documentation go through all the steps to host your website on the blockchain and register
-it on Massa's DNS service in order to access it using Massa's plugin.
+This documentation go through all the steps to host your website on the blockchain and register it on Massa’s DNS service in order to access it using any classic browser.
 
-To upload a website to Massa's decentralized web you can use the following smart-contract.
-If you want latter want to be able to access to your website through the Massa plugin it's
-important to use the entry `massa_web` for your website.
+To do so you will need to follow 3 steps: 
 
-.. code-block:: typescript
+1. Install Thyra on your machine
+2. Create a wallet
+3. Define your DNS & upload your website on the blockchain
 
-    import { include_base64, print, Storage } from "massa-sc-std";
+For each of the steps above, you will have the choice to do it manually or automatically via Thyra.
 
-    function createWebsite(): void {
-        const bytes = include_base64('./site.zip');
-        Storage.set_data("massa_web", bytes);
-    }
 
-    export function main(_args: string): i32 {
-        createWebsite();
-        print("Uploaded site");
-        return 0;
-    }
+*What is Thyra*
+===============
 
-This smart-contract and the dependencies are available `here <https://github.com/massalabs/massa-sc-examples/tree/main/website>`_.
+θύρα to pronounce thùra and to write Thyra is way simpler that his name looks like! It means entrance, gateway in ancient greek. Thyra is thus a gateway to Massa blockchain.
```

Comment:
You probably right. @gregLibert do you agree with Ena's comment? 
I think I like this story too much so I'm putting it everywhere, but if you both agree, let's just delete this section.

Reply:
I think that the name we give to things is extremely important since it allows people to "project" themselves onto the named object.

I am of the opinion to remove this paragraph and add a "Why this name" section.
For this section I imagine something like:

As explained Thyra plays the role of a gateway to the Massa blockain. Massa being a coin minted in the city of [massalia](https://en.wikipedia.org/wiki/Massalia), the ancient marseille founded by the greek, quite naturaly the name [θύρα](https://en.wiktionary.org/wiki/%CE%B8%CF%8D%CF%81%CE%B1) (entrance, front door in ancient greek) imposed itself.
In order to simplify and standardize its writing, we transformed it into Thyra, but its pronunciation [\\tʰý.raː\\](https://www.youtube.com/watch?v=_0BQ7sSJMTw) remained.

Reply:
Thank you guys. I understand your points, i've updated the doc accordingly.